test/renderer/agendas/index_spec.js

```
@@ -5,7 +5,8 @@ import { liveStore, dispatchQueuePromise, waitForOutputs } from '../../utils';
 
 describe('agendas.executeCell', function() {
   this.timeout(5000);
-  it('produces the right output', () => {
+  it('produces the right output', (done) => {
+    setTimeout(done, 5000);
```

Ah, it's similar to a bug I just dealt with.  This same test copy and pasted into a test folder called "widgets" would fail because it was being executed after the "components" tests.  The trick in that case was that the bug was in the component tests, specifically with the Latex component.  I added a global flag to disable MathJax per test and that seemed to do the trick.  Maybe the timeout is something similar?  Have you tried making the timeout more verbose?  It may be eating an error which is causing the timeout...